lib/web_ui/lib/src/engine/app_bootstrap.dart

```diff
@@ -40,33 +40,26 @@ class AppBootstrap {
       // This is a convenience method that lets the programmer call "autoStart"
       // from JavaScript immediately after the main.dart.js has loaded.
       // Returns a promise that resolves to the Flutter app that was started.
-      autoStart: allowInterop(() => futureToPromise(() async {
+      autoStart: (() => futureToPromise(() async {
         await autoStart();
         // Return the App that was just started
-        return _prepareFlutterApp();
-      }())),
+        return _prepareFlutterApp() as JSAny;
```

Comment:
nit: This (and similar casts elsewhere) is unneeded if the return type is a `@staticInterop` type.

Reply:
Nevermind, `futureToPromise` returns a `T extends JSAny` so this is fine.

Reply:
Looks like I could solve this by making `FlutterApp` and `FlutterAppRunner` extend `JSObject`.